easybuild/easyconfigs/c/CUnit/CUnit-2.1-3-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'CUnit'
+version = '2.1-3'
```

Comment:
@iotaka version here conflicts with what's used in https://github.com/easybuilders/easybuild-easyconfigs/pull/16972 ?

Reply:
typo in the title, fixed